addon/v1alpha1/types_addonhubconfig.go

```diff
@@ -0,0 +1,50 @@
+package v1alpha1
```

Comment:
for addonhubconfig, I think we should put in a separated PR

Reply:
removed